Documentation/helm-ceph-cluster.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Ceph Cluster
+weight: 10200
+indent: true
+---
+
+{% include_relative branch.liquid %}
+
+# Ceph Cluster Helm Chart
+
+Installs a [Ceph](https://ceph.io/) cluster on Rook using the [Helm](https://helm.sh) package manager.
+
+## Warning
+
+This chart is released to the community for testing but is considered in development. It is currently only available on the `master` channel.
+Once released, the repository should be updated to the `release` channel if you wish to switch to the stable branch.
```

Comment:
Helm will allow changing the channel, right? Hopefully we can confirm this won't be an issue later.
```suggestion
Once released (targeted for v1.7), the repository should be updated to the `release` channel at that time for the stable branch.
```